deep_qa/run.py

```diff
@@ -0,0 +1,137 @@
+import sys
+import logging
+import shutil
+
+import random
+import pyhocon
+import numpy
+
+# pylint: disable=wrong-import-position
+from deep_qa.common.params import Params, replace_none, ConfigurationError
+from deep_qa.common.tee_logger import TeeLogger
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def prepare_environment(params: Params):
+    seed = params.pop("random_seed", 13370)
+    numpy_seed = params.pop("numpy_seed", 1337)
+    if "keras" in sys.modules:
+        logger.warning("You have already imported Keras in your code. If you are using DeepQA"
+                       "functionality to set random seeds, they will have no effect, as code"
+                       "prior to calling this function will be non-deterministic. We will not"
+                       "the random seed here.")
+        seed = None
+        numpy_seed = None
+    if seed is not None:
+        random.seed(seed)
+    if numpy_seed is not None:
+        numpy.random.seed(numpy_seed)
+
+    from deep_qa.common.checks import log_keras_version_info
+    log_keras_version_info()
+
+
+def train_model(param_path: str, model_class=None):
+    """
+    This function is the normal entry point to DeepQA. Use this to run a DeepQA model in
+    your project. Note that if you care about exactly reproducible experiments,
+    you should avoid importing Keras before you import and use this function, as
+    Keras relies on random seeds which can be set in this function via a
+    JSON specification file.
+
+    Parameters
+    ----------
+    param_path: str, required. A json file specifying a DeepQaModel.
+    model_class: A DeepQaModel, optional. This option is useful if you have
```

Comment:
Same here, and it should just be: `model_class: DeepQaModel, optional (default=None)` on the first line.